Cel 23 update documentation post unit integration (#21)
JaiWillems/Celest: docs/source/modules/units.rst

Units
=====

.. contents:: Contents
   :depth: 1
   :local:

Units Overview
--------------

Celest provides the ability to handle different units of measurement and is accomplished through the use of units an
quantities. The following sections describe the different units available, how to use them, and how to convert between
them via the :class:`Quantity` class.

Handling Units
--------------

Initialized units are accessed from the units module which can be imported by the following:

.. code-block:: python

   from celest import units as u

> The primary units can then be accessed using a dot notation.

Author: ```suggestion
Primary units can then be accessed using a dot notation.
```

.. code-block:: python

   u.m  # Meter measure.
   u.s  # Second measure.
   u.deg # Degree measure.

> The following table lists the primary units, their code use label, and their measure type.

Author: ```suggestion
The following table lists all primary units, their code call sign, and their measure type.
```

.. list-table:: Available Units
   :widths: 30, 30, 30
   :header-rows: 1

   * - Unit
     - Code Label
     - Type
   * - Millimeter
     - `u.mm`
     - Length
   * - Centimeter
     - `u.cm`
     - Length
   * - Meter
     - `u.m`
     - Length
   * - Kilometer
     - `u.km`
     - Length
   * - Inch
     - `u.inch`
     - Length
   * - Feet
     - `u.ft`
     - Length
   * - Yard
     - `u.yd`
     - Length
   * - Mile
     - `u.mi`
     - Length
   * - Second
     - `u.s`
     - Time
   * - Minute
     - `u.min`
     - Time
   * - Hour
     - `u.hr`
     - Time
   * - Day
     - `u.dy`
     - Time
   * - JD2000
     - `u.jd2000`
     - Date
   * - Degree
     - `u.deg`
     - Angle
   * - Radian
     - `u.rad`
     - Angle
   * - Seconds of Arc
     - `u.arcsec`
     - Angle
   * - Minutes of Arc
     - `u.arcmin`
     - Angle
   * - Hour Angle
     - `u.hourangle`
     - Angle

Primary units can be combined with others through multiplication or division to create compound units.

.. code-block:: python

   velocity_unit = u.m / u.s  # Meter per second.
   acceleration_unit = u.m / u.s ^ 2  # Meter per second squared.

> Units are typically passed into Celest's class constructors, methods, and functions as a unit parameter to specify the

Author: ```suggestion
Units are typically passed into Celest's class constructors, methods, and functions as a parameter to specify the
```
